ui/src/pages/frontsite/PatronProfile/PatronPendingLoans/PatronPendingLoans.js

```diff
@@ -44,6 +44,15 @@ class LoanRequestListEntry extends Component {
               <Item.Meta>
                 <DocumentAuthors metadata={loan.metadata.document} />
                 Requested on {toShortDate(loan.metadata.request_start_date)}
+                <br />
+                Valid until {toShortDate(loan.metadata.request_expire_date)}
+                <Popup
+                  content={
+                    'If the request was not processed ' +
```

Comment:
I think it is bad if we show to the user this: he might thinks that the librarian do not want to process his/her request! :)
Maybe better to highlight this in the backoffice, and not in the frontsite!

Reply:
That is existing code from previous, I just moved it on the left under the `Requested on` information so we can accommodate the cancel button on the right. I am sure @kprzerwa has a better idea what to do with it :D

Reply:
Sure, not saying that you added it, I was just "thinking out loud" when I saw it! :) :)

Reply:
I think that leaving this without the information will be confusing, as the request will just disappear  from the profile page of the user without giving him a chance to act upon the request which is not processed by the librarian.
In that case we should probably show the expired requests too and have a feature of "request again" or similar. We could discuss it in person.